episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+images. As there is one nucleus per cell, we will use the nucleus number as the 
+equivalent of cell number. If we look at our cells image in Napari, we can 
+clearly see individual nuclei highlighted in green. If we zoom in we can see 
+some noise (as a slight 'graininess' over the image), but this doesn't interfere 
+with being able to see the locations or sizes of all nuclei.
+
+Next, it's good practice to check the image histogram, as we covered in the 
+[choosing acquisition settings episode
+](choosing-acquisition-settings.md#initial-quality-control). We'll do this in 
+the exercise below:
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Histogram quality control
+
+Plot the image histogram with `napari matplotlib`:
+
+- Do you see any issues with the histogram?
+
+If you need a refresher on how to use `napari matplotlib`, check out the 
+[image display episode](image-display.md#napari-plugins).
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+In the top menu bar of Napari select:  
+`Plugins > napari Matplotlib > Histogram`
+
+Make sure you have the 'nuclei' layer selected in the layer list (should be 
+highlighted in blue).
+
+![](fig/nuclei-histogram.png){alt="A histogram of the 29th z slice of Napari's 
+cell sample image"}
+
+Note that as this image is 3D `napari matplotlib` only shows the histogram of 
+the current z slice (in this case z=29 as shown at the top of the histogram). 
+Moving the slider at the bottom of the viewer, will update the histogram to show 
+different slices.
+
+If we look at the brightest part of the image, near z=29, we can see that there 
+are pixel values over much of the possible range. This being said, it is a bit 
+underexposed as pixel values are clustered at the left hand side, with few 
+values to the right. In an ideal world the image could have been acquired with a 
+longer exposure time / higher laser power etc to collect more light. This slight 
+underexposure isn't a big issue here though, as we can still clearly separate 
+the nuclei from the background which is our main aim.
+
+Most importantly, we see no evidence for 'clipping', which means we are avoiding 
+any irretrievable loss of information this would cause. Recall that clipping 
+occurs when pixels are recording light above the maximum limit for the image. 
+Therefore, many values are 'clipped' to the maximum value resulting in 
+information loss.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## What is segmentation?
+
+In order to count the number of cells, we must 'segment' the nuclei in this 
+image. Segmentation is the process of figuring out what each pixel in an image 
+represents e.g. is that pixel part of a nucleus or not? Segmentation comes in 
+two main types - 'semantic segmentation' and 'instance segmentation'. 
+
+Let's take a quick look at a rough semantic segmentation. Open Napari's console 
+by pressing the ![](
```

Comment:
I think elsewhere we've said use window -> console from the top menu. Should we be consistent? Or is it better to give different options?

Reply:
As far as I can see, we always use the console button at the bottom left of Napari. Do let me know if you find an inconsistency in another episode though!

Reply:
Maybe I'm getting confused with something else (like opening files from the menu).